### DISCUSSION TITLE
Documentation Bug: Publishing Config Does Not Work

### DISCUSSION THREAD

**Discussion by fkellner** [spatie/laravel-markdown#2] [Q&A]
Dear Spatie Team,

As documented on [this page](https://spatie.be/docs/laravel-markdown/v1/installation-setup), I ran `php artisan vendor:publish --provider="Spatie\LaravelMarkdown\LaravelMarkdownBladeComponentServiceProvider" --tag="markdown-config"` in order to be able to globally edit the configuration.

Artisan was, however, `Unable to locate publishabel resources.`.

Thanks to a little research, I found the correct command:
`php artisan vendor:publish --provider="Spatie\LaravelMarkdown\MarkdownServiceProvider" --tag="markdown-config"`.

Please update your documentation.

Best Regards

P.S.: I do not seem to be able to create issues in your project, what's up with that?

**Comment by freekmurze** [comment]
Nice catch, feel free to submit a PR to the readme that fixes this 👍

On not having an issues tab: https://github.com/alpinejs/alpine/discussions/1423

**Reply by fkellner** [reply]
https://github.com/spatie/laravel-markdown/pull/6
